### ISSUE TITLE
vite-imagetools doesn't pick up my images on build (SvelteKit + MDsveX)

### ISSUE THREAD

**Issue by wentallout** [JonasKruckenberg/imagetools#398]
I was using [MDsveX](https://github.com/pngwn/MDsveX) to render a blog page but the images I import into it don't work when transformed into webp using imagetools.

My code on the markdown page
```
import thumbnail from '$lib/assets/images/project/reza/reza-thumbnail.png?webp';

```

This is how it looks on `dev`

![photo_2022-09-20_11-58-00](https://user-images.githubusercontent.com/76118931/191672004-cda5841a-5e9b-4ea7-8b5e-37d0486dbe3a.jpg)

This is the same page on `preview`

![photo_2022-09-20_12-08-29](https://user-images.githubusercontent.com/76118931/191672223-1a509afd-fa80-4224-98b9-11c76e198873.jpg)

**Comment by AlexDaichendt** [comment]
Problem is - as far as I can tell - related to the path of the image in production. If the current page is two folders deep, `./_app` will incorrectely try to fetch the image from `root.com/path/_app` instead of `root.com/_app`.

**Comment by benmccann** [comment]
Can you check if https://github.com/JonasKruckenberg/imagetools/pull/400 would fix the issue you're facing?

**Comment by wentallout** [comment]
> Can you check if #400 would fix the issue you're facing?

I'll take a look but I feel like I should wait for the PR/update to finish.

**Comment by wentallout** [comment]
> Can you check if #400 would fix the issue you're facing?

hey, Is the fix still being worked on? My project is stuck because my images route is still wrong on build. Can you let me know if that pull request is approved?

**Comment by benmccann** [comment]
https://github.com/JonasKruckenberg/imagetools/pull/400 was merged in https://github.com/JonasKruckenberg/imagetools/pull/412 and released in 4.0.10
